exercises/space-age/canonical-data.json

```diff
@@ -3,22 +3,22 @@
   "version": "1.3.0",
```

Comment:
Tests are being added, renamed, or removed, so we'll need to bump the minor version number.

`"version": "1.4.0",`

Reply:
bump it twice?

Reply:
Ah, no, don't do that. I didn't realize I was only looking at the changes for a single commit, not the whole PR. You're good already.